class BaseUserException(Exception):
    pass


class UserNotFound(BaseUserException):
    pass
